src/gui/wizard/owncloudadvancedsetuppage.cpp

```diff
@@ -200,6 +215,11 @@ QStringList OwncloudAdvancedSetupPage::selectiveSyncBlacklist() const
     return _selectiveSyncBlacklist;
 }
 
+bool OwncloudAdvancedSetupPage::confirmBigFolder() const
```

Comment:
Would be good to have such functions named "is..." or "are..." to see it is a getter